Stylesheets/components/cart-page.css

```diff
@@ -0,0 +1,26 @@
+.cart-quantity{
+margin-top: -2rem;
+}
+
+.quantity-header{
+color:var(--white-color)
+}
+
+.quant-input{
+width:1rem;
+
+}
+
+.check-out-text{
+    font-size: 1rem;
+    margin:0
+}
+
+.checkout-gap{
+    margin:1.5rem
```

Comment:
You can format your CSS using prettier. Sometimes there is a space after the colon in your rules, other times no space.